server/test/test_feature_pipeline_augmentations.py

```
@@ -0,0 +1,82 @@
+
+def test_resample_window_handles_single_target_and_empty_input_guard():
+    feature_pipeline = importlib.import_module("amyserver_tools.feature_pipeline")
+
+    window = np.ones((4, 3), dtype=np.float32)
+    resampled = feature_pipeline._resample_window(window, 1)
+    assert resampled.shape == (1, 3)
+
+    empty_window = np.zeros((0, 3), dtype=np.float32)
+    try:
+        feature_pipeline._resample_window(empty_window, 1)
+        assert False, "Expected ValueError for empty window"
```

Ruff (0.15.2)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)